src/liquid/components/ld-button/ld-button.tsx

```diff
@@ -143,7 +138,8 @@ export class LdButton implements InnerFocusable, ClonesAttributes {
     this.el.removeEventListener('click', this.handleClick, {
       capture: true,
     })
-    this.attributesObserver?.disconnect()
+    /* istanbul ignore if */
+    if (this.attributesObserver) this.attributesObserver.disconnect()
```

Comment:
No, this is not what I'm doing here. The `if` part is new, and actually fixes the issue described in the PR / the referenced issue on GitHub.

For other parts in this PR I may have changed optional chaining to explicit if statements in order for istanbul to ignore a specific branch, which I think is acceptable, because I didn't change any logic there. But here the change in logic is intentional and has nothing to do with test coverage.